<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderSurface</class>
 <widget class="QWidget" name="RenderSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="configGroupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Config</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="2" column="1" colspan="3">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="6" rowspan="2">
           <widget class="QGroupBox" name="cubeAxesGroupBox">
            <property name="title">
             <string>Cube Axes</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_15">
             <item row="1" column="0">
              <layout class="QGridLayout" name="gridLayout_14">
               <item row="5" column="0">
                <widget class="QCheckBox" name="showZAxisCheckBox">
                 <property name="text">
                  <string>Show Z Axis</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Fly Mode</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Color</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="Widgets::ColorPushButton" name="cubeAxesColorButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="numOfLabelsSpinBox">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Number of Labels</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="flyModeComboBox">
                 <item>
                  <property name="text">
                   <string>Outer Edge</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Closest Triad</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QCheckBox" name="showXAxisCheckBox">
                 <property name="text">
                  <string>Show X Axis</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QCheckBox" name="showYAxisCheckBox">
                 <property name="text">
                  <string>Show Y Axis</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="xLabelEdit">
                 <property name="placeholderText">
                  <string>X Label</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="yLabelEdit">
                 <property name="placeholderText">
                  <string>Y Label</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="zLabelEdit">
                 <property name="placeholderText">
                  <string>Z Label</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="5" rowspan="2">
           <widget class="QGroupBox" name="contourGroupBox">
            <property name="title">
             <string>Contour</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="4" column="0" colspan="2">
                <widget class="QCheckBox" name="showLabelsCheckBox">
                 <property name="text">
                  <string>Show Labels</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Lines Color</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="Widgets::ColorPushButton" name="contourLinesColorButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="Widgets::ColorPushButton" name="labelsColorButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Labels Color</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Number of Contours</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="numOfContoursSpinBox"/>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="showContourLinesCheckBox">
                 <property name="text">
                  <string>Show Contour Lines</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3" rowspan="2">
           <widget class="QGroupBox" name="outlineGroupBox">
            <property name="title">
             <string>Outline</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="1">
                <widget class="Widgets::ColorPushButton" name="outlineButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Outline Color</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="2">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="outlineOpacitySpinBox">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Opacity</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="genFacesCheckBox">
                 <property name="text">
                  <string>Generate Faces</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Line Width</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="outlineLineWidthSpinBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QGroupBox" name="scalarBarGroupBox">
            <property name="title">
             <string>Scalar Bar</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="2" column="1">
                <widget class="QCheckBox" name="showColorBarCheckBox">
                 <property name="text">
                  <string>Show Color Bar</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="numOfColorsSpinBox"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Number of Colors</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QCheckBox" name="showTickLabelsCheckBox">
                 <property name="text">
                  <string>Show Tick Labels</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="Widgets::ColorPushButton" name="frameColorButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="showFrameCheckBox">
                 <property name="text">
                  <string>Show Frame</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Orientation</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="scalarBarOrientationComboBox">
                 <item>
                  <property name="text">
                   <string>Horizontal</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Vertical</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Frame Color</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Frame Width</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QSpinBox" name="frameWidthSpinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="tableGroupBox">
            <property name="title">
             <string>Table</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_17">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_16">
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Rows</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="rowsSpinBox">
                 <property name="minimum">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QPushButton" name="randomButton">
                 <property name="text">
                  <string>Random</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QPushButton" name="loadFileButton">
                 <property name="text">
                  <string>Load File</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QGroupBox" name="generalGroupBox">
            <property name="title">
             <string>General</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="6" column="1">
                <widget class="Widgets::GradientComboBox" name="gradientCombo"/>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Gradient</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="Widgets::ColorPushButton" name="bgColorButton">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="sourcesButton">
                 <property name="text">
                  <string>Sources</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Bg Color:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Edge Color</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="viewFitButton">
                 <property name="text">
                  <string>View Fit</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="Widgets::ColorPushButton" name="edgeColorButton">
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QCheckBox" name="showEdgeCheckBox">
                 <property name="text">
                  <string>Show Edge</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="showOriMarkerButton">
                 <property name="text">
                  <string>Show Orientation Marker</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Data</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTableView" name="tableView">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Preview</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="previewLayout"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Widgets::GradientComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/gradientcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
